data/migrations/deb/x_x_xx_to_y_y_yy.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/python3.5
+import os
+import shutil
+import glob
+from importlib.util import module_from_spec, spec_from_file_location
+
+from stp_core.common.log import getlogger
+
+baseDir = '/var/lib/indy'
+GENERAL_CONFIG_DIR = '/etc/indy'
+GENERAL_CONFIG_FILE = 'indy_config.py'
+LOG_DIR = '/var/log/indy'
+NODE_BASE_DATA_DIR = baseDir
+CLI_BASE_DIR = '/home/indy/.indy-cli'
+CLI_NETWORK_DIR = '/home/indy/.indy-cli/networks'
+
+logger = getlogger()
+
+old_base_dir = '/home/indy/.indy'
+
+
+def migrate_genesis_txn(old_base_txn_dir, new_base_txn_dir):
+    logger.info('Move genesis transactions {} -> {}'.format(
+        old_base_txn_dir, new_base_txn_dir))
+    for suffix in ('sandbox', 'live', 'local'):
+        new_txn_dir = os.path.join(new_base_txn_dir, suffix)
+        os.makedirs(new_txn_dir, exist_ok=True)
+
+        old_domain_genesis = os.path.join(
+            old_base_txn_dir, 'domain_transactions_{}_genesis'.format(suffix))
+        old_pool_genesis = os.path.join(
+            old_base_txn_dir, 'pool_transactions_{}_genesis'.format(suffix))
+        new_domain_genesis = os.path.join(
+            new_txn_dir, 'domain_transactions_genesis')
+        new_pool_genesis = os.path.join(
+            new_txn_dir, 'pool_transactions_genesis')
+
+        if os.path.exists(old_domain_genesis):
+            shutil.move(old_domain_genesis, new_domain_genesis)
+        if os.path.exists(old_pool_genesis):
+            shutil.move(old_pool_genesis, new_pool_genesis)
+
+    logger.info('done')
+
+
+def migrate_keys(old_base_keys_dir, new_base_keys_dir):
+    for prefix in ('private', 'public', 'sig', 'verif'):
+        old_keys_dir = os.path.join(old_base_keys_dir, '{}_keys'.format(prefix))
+        if os.path.exists(old_keys_dir) and os.path.isdir(old_keys_dir):
+            shutil.move(old_keys_dir, new_base_keys_dir)
+
+
+def get_network_name():
+    network_name = 'sandbox'
+    old_general_config = os.path.join(old_base_dir, 'indy_config.py')
+    spec = spec_from_file_location('old_general_config', old_general_config)
+    old_cfg = module_from_spec(spec)
+    spec.loader.exec_module(old_cfg)
+    if hasattr(old_cfg, 'current_env') and old_cfg.current_env != 'test':
+        network_name = old_cfg.current_env
+    return network_name
+
+
+def migrate_general_config(old_general_config, new_general_config, network_name):
+    logger.info('Migrate general config file {} -> {} for network {}'.format(
+        old_general_config, new_general_config, network_name))
+    f = open(old_general_config, "r")
+    lines = f.readlines()
+    f.close()
+
+    f = open(new_general_config, "w")
+    for line in lines:
+        if not line.startswith('current_env'):
```

Comment:
We should also replace 
```
poolTransactionsFile = 'pool_transactions_live'
domainTransactionsFile = 'domain_transactions_live'
```
to NETWORK_NAME. So, these lines should also be removed from config files.

Reply:
done